examples/vision/depth_estimation.py

```diff
@@ -105,7 +106,7 @@
 """
 
 
-class DataGenerator(tf.keras.utils.Sequence):
+class DataGenerator(keras.utils.Sequence):
```

Comment:
You can use the up-to-date `PyDataset` class here

Reply:
Modified to PyDataset.